docs/diagrams/SaveSequenceDiagram.puml

```diff
@@ -0,0 +1,29 @@
+@startuml
```

Comment:
Misleading Usage of activation blocks. Should'nt UI still be activated after calling executeCommand() and until it recieves a return value from MeetingJio?

Reply:
![image](https://user-images.githubusercontent.com/77721568/160958393-5c4f0259-b5e6-43f2-a9ab-4b2ab17f924d.png)